README.md

```diff
@@ -73,145 +72,101 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Fixed an issue causing the Settings to not be saved correctly on scenarios where the admin user would be different then the user running PowerToys.
+- Added a setting to pick which language should be picked by default for Text Extractor.
+
+### Text Extractor
 
-### Shortcut Guide
-- Added a setting to make the shortcuts and taskbar icons have different configurable response times. Thanks [@OkamiWong](https://github.com/OkamiWong)!
+- Improve behavior for CJK languages by not adding spaces for some characters that don't need it. Thanks [@AO2233](https://github.com/AO2233)!
+- OCR language can now be picked in the right-click context menu.
 
 ### Video Conference Mute
 
-- Changed the warning about deprecating Video Conference Mute to saying it's going to go into legacy mode, thanks to community feedback.  (This was a hotfix for 0.64)
+- Reduced resource consumption by not starting the File Watchers when the utility is disabled.
 
 ### Documentation
 
-- Added the core team to COMMUNITY.md
+- Updated the development setup documentation.
+- Improved the Markdown documentation lists numbering in many docs. Thanks [@sanidhyas3s](https://github.com/sanidhyas3s)!
 
 ### Development
 
-- Fixed some errors in the GitHub issue templates. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
-- Updated the Windows implementation library. Thanks [@AtariDreams](https://github.com/AtariDreams)!
-- Added Hosts File Editor to the issue templates. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 - Turned on C++ code analysis and incrementally fixing warnings.
-- Cleaned up unused dependencies. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
-- Fixed building on the latest MSVC.
-- Fixed multi-processor build on the latest MSBuild.
-- Added a message to suggest the feedback hub to the fabric bot triggers.
-- Optimized every png file with the zopfli algorithm. Thanks [@pea-sys](https://github.com/pea-sys)!
-- Updated the .vsconfig file for a quicker development setup. Thanks [@ChaseKnowlden](https://github.com/ChaseKnowlden)!
+- C++ code analysis no longer runs on release CI to speed up building release candidates. It still runs on GitHub CI and when building locally to maintain code quality.
+- Cleaned up "to-do" comments referring to disposing memory on C#. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Added a fabric bot rule for localization issues.
+- Fixed a CI build error after a .NET tools update.
+- Update the windows app sdk dependency version to 1.2.
+- When building for arm64, the arm64 build tools are now preferred when building on an arm64 device. Thanks [@snickler](https://github.com/snickler)!
+- Updated the C# test framework and removed unused Newtonsoft.Json package references.
+- Updated StyleCop and fixed/enabled more warnings. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 - Fixed a language typo in the code. Thanks [@eltociear](https://github.com/eltociear)!
```

Comment:
Same fix in two versions?

Reply:
A different typo, but still the same thing.

Reply:
Thanks for all the feedback!